src/cordova/android/OutlineAndroidLib/outline/src/main/aidl/org/outline/TunnelConfig.aidl

```diff
@@ -14,10 +14,8 @@
 
 package org.outline;
 
-import org.outline.shadowsocks.ShadowsocksConfig;
-
 parcelable TunnelConfig {
```

Comment:
I'm, not convinced we need this type, versus passing the values that are needed explicitly in the different points. The extra type adds an intermediate step that sometimes hurts readability, especially since not all fields are needed everywhere this type is being used.